### DISCUSSION TITLE
Increase members' community contribution

### DISCUSSION THREAD

**Discussion by qianxliu** [zoo-js/zoo#183] [General] [answered]
使用General是因为这是一个General问题。当前社区有个很大的问题就是member贡献几乎为0。我认为必须要有一些方案增加member对社区的贡献，这样社区才能活跃——这样社区才不是仅仅两三人的社区。但我没有什么切实的提议，因为如果活跃一些工程的话，可能这些工程的门槛还不能太高。

**Comment by xrkffgg** [comment, marked as answer]
暂时也没想到很好的方式
